src/linux_usbmon.pyx

```diff
@@ -76,6 +232,19 @@ def monitor(fid):
     if usbmon == MAP_FAILED:
         raise OSError(errno, 'mmap failed')
 
+    print('\n'.join([
+        # logo source: http://patorjk.com/software/taag/#p=display&f=Rectangles&t=cython%20linux%20usbmon%20monitor
+        "",
+        "             _   _              _ _                        _                                  _ _           ",
+        "     ___ _ _| |_| |_ ___ ___   | |_|___ _ _ _ _    _ _ ___| |_ _____ ___ ___    _____ ___ ___|_| |_ ___ ___ ",
+        "    |  _| | |  _|   | . |   |  | | |   | | |_'_|  | | |_ -| . |     | . |   |  |     | . |   | |  _| . |  _|",
+        "    |___|_  |_| |_|_|___|_|_|  |_|_|_|_|___|_,_|  |___|___|___|_|_|_|___|_|_|  |_|_|_|___|_|_|_|_| |___|_|  ",
+        "        |___|                                                                                               ",
+        "",
+        "    Example USB monitor implemented with cython-linux-usbmon",
```

Comment:
Probably worth removing from here — beside not actually reading in the code, it would make the output fairly verbose on the command line.

But just so you know, you can get a more readable version of the same by using:

```
print("""\
                                                                                                        
         _   _              _ _                        _                                  _ _           
 ___ _ _| |_| |_ ___ ___   | |_|___ _ _ _ _    _ _ ___| |_ _____ ___ ___    _____ ___ ___|_| |_ ___ ___ 
|  _| | |  _|   | . |   |  | | |   | | |_'_|  | | |_ -| . |     | . |   |  |     | . |   | |  _| . |  _|
|___|_  |_| |_|_|___|_|_|  |_|_|_|_|___|_,_|  |___|___|___|_|_|_|___|_|_|  |_|_|_|___|_|_|_|_| |___|_|  
    |___|                                                                                               
""")
```

If you want it to indent you can wrap the triple-quotes in `textwrap.dedent()`.